HacktoberfestProject.Web/Data/Entities/PrEntity.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace HacktoberfestProject.Web.Models.Entities
+﻿namespace HacktoberfestProject.Web.Data.Entities
```

Comment:
Why the move to Data from Models?

Reply:
Was purely to keep all database items together. I'll shift them to the models directory.